code/game/machinery/doors/poddoor.dm

```diff
@@ -43,7 +43,7 @@
 
 /obj/machinery/door/poddoor/crowbar_act(mob/living/user, obj/item/tool)
 	. = ..()
-	if (density)
+	if (density & hasPower())
```

Comment:
```suggestion
	if (density && hasPower())
```

The better way to fix this would be to move opening airlocks to `door/crowbar_act`, then check `if (.) return`, but this should work fine for now.

Reply:
I'll try to do that.